.gitignore

```diff
@@ -16,6 +16,9 @@ ipyvuetify/labextension/
 ipyvuetify/nbextension/
 js/lib
 
-#readthedocs
+# readthedocs
 _build/
 
+# vs code 
+.vscode/
```

Comment:
This is not related to using pre-commit. Also, we want to limit this to files specific to the project,  not to the environment of the developer. The global .gitignore can be used for this (in [home]/.gitignore).

Reply:
that's funny I had the exact same argument few weeks ago in the commitizen repository https://github.com/commitizen-tools/commitizen/pull/657 but they were using a broader .gitignore file so I think I can't win here ;-)

Reply:
I see you won there :-). It's indeed a preference, but don't you think having `.vscode` (or `.idea` in my case) in `~/.gitignore` is a good solution for developer specific files? The developer sets this up once for their system, then it works for all projects.